fastq_to_TMB_processes.nf

```diff
@@ -140,8 +139,6 @@ process trimPair {
 
 process alignReads {
     tag "$an_id"
-    cpus 48
```

Comment:
Needs to be parameterized in the `-t 48` string in the bwa command

Reply:
Fixed.